### PR TITLE
Ad fmcjesdadc1 ebz fix

### PR DESCRIPTION
**Pull request by cristiantpop**
Fix SPI init value for spi_init.type
This parameter is platform dependent and some compiler switch has been
added.

### CODE REVIEW

File: ad-fmcjesdadc1-ebz/ad9250.c [hunk 1]
```diff
@@ -105,6 +105,10 @@ int32_t ad9250_setup(struct ad9250_dev **device,
 
     /* Setup SPI descriptor */
     ret = spi_init(&dev->spi_dev, &init_param.spi_init);
+	if(ret < 0) {
+		ad_printf("Error spi_init()\n");
```
Comment (mirceacaprioru) [inline ad-fmcjesdadc1-ebz/ad9250.c]: if you just want to print the function name you can use __func__ if the ad_printf accepts parameters like %s

ad_printf("Error %s\n", __func__);

File: ad-fmcjesdadc1-ebz/ad_fmcjesdadc1_ebz.c [hunk 2]
```diff
@@ -88,9 +88,29 @@ int main(void)
 	rx_xfer_1.start_address = XPAR_DDR_MEM_BASEADDR + 0x900000;
 #endif
 
+#ifdef ZYNQ_PS7
```
Comment (mirceacaprioru) [inline ad-fmcjesdadc1-ebz/ad_fmcjesdadc1_ebz.c]: we should try to keep these compiler switches to a minimum. If someting can be done without them then use that option. I'm not sure in this case what could be done.

**Comment by cristiantpop** [comment]
v2:
Fix travis errors

File: ad-fmcjesdadc1-ebz/ad9250.c (hunk 1 above)
Comment (cristiantpop) [reply ad-fmcjesdadc1-ebz/ad9250.c]: In this case the current function is not spi_init but ad9250_setup, but the error is from SPI

**Comment by mirceacaprioru** [comment]
will merge later today

**Pull request merged**